src/api/test/unit/project_test.rb

```diff
@@ -372,6 +372,29 @@ def test_handle_project_links
     project_b.destroy
   end
 
+  def test_project_kind_removal
+    User.session = users(:king)
+
+    prj = Project.new(name: 'kind')
+    prj.update_from_xml!(Xmlhash.parse(
+                           "<project name='kind' kind='maintenance_release'>
+                             <title/>
+                             <description/>
+                           </project>"
+                         ))
+
+    xml = prj.to_axml
+    assert_xml_tag xml, tag: :project, attributes: { kind: 'maintenance_release' }
+
+    prj.update_from_xml!(Xmlhash.parse(
+                           "<project name='kind'>
+                             <title/>
+                             <description/>
+                           </project>"
+                         ))
+    assert_no_xml_tag xml, tag: :project, attributes: { kind: 'maintenance_release' }
```

Comment:
```suggestion
    prj.update_from_xml!(Xmlhash.parse(
                           "<project name='kind'>
                             <title/>
                             <description/>
                           </project>"
                         ))
    xml = prj.to_axml
    assert_no_xml_tag xml, tag: :project, attributes: { kind: 'maintenance_release' }
    assert_xml_tag xml, tag: :project, attributes: { kind: 'standard' }
```

If removing the kind of a project makes its new kind to be `'standard'`, shouldn't we check for standard too?

Reply:
The minitests fail. I think there is an assignment missing in the test:

```
xml = prj.to_axml
```